src/main/java/uk/gov/indexer/ctserver/CTServer.java

```diff
@@ -0,0 +1,86 @@
+package uk.gov.indexer.ctserver;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.Response;
+import java.io.ByteArrayOutputStream;
+import java.security.MessageDigest;
+import java.util.Base64;
+
+public class CTServer {
```

Comment:
Mentioned in slack but recording here for posterity: there already is a [java ct client](https://github.com/google/certificate-transparency/blob/master/java/src/org/certificatetransparency/ctlog/comm/HttpLogClient.java) but it's not clear how to build it with the gclient workflow and I don't think any jars are published for it.